fastlane/Fastfile

```diff
@@ -14,9 +14,6 @@ APP_STORE_METADATA_FOLDER = File.join(FASTLANE_FOLDER, 'metadata')
 ARTIFACTS_FOLDER = File.join(PROJECT_ROOT_FOLDER, 'artifacts')
 SECRETS_FOLDER = File.join(Dir.home, '.configure', 'pocketcasts-ios', 'secrets')
 VERSION_XCCONFIG_PATH = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.xcconfig')
-# Unfortunately, ios_current_branch_is_hotfix relies on this ENV var under the hood.
-# We can't get rid of it just yet.
-ENV['PUBLIC_CONFIG_FILE'] = VERSION_XCCONFIG_PATH
```

Comment:
A quick search of `PUBLIC_CONFIG_FILE` in the source code of `release-toolkit` confirms that this ENV var is not referenced by any other action from `release-toolkit` anymore.

A quick search of `PUBLIC_CONFIG_FILE` in this repo also confirms that this ENV var doesn't seem to be used by the `Fastfile` directly either.

So I'm pretty sure it's safe to remove now.